site/installing-weave.md

```diff
@@ -28,6 +28,18 @@ With Weave Net downloaded onto your VMs or hosts, you are ready to launch a Weav
   <img src="hello-screencast.png" alt="Click to watch the screencast" />
 </a>
 
+###Checkpoint
+
+Weave Net [periodically contacts Weaveworks servers for available
+versions](https://github.com/weaveworks/go-checkpoint).  New versions
+are announced in the log and in [the status
+summary](/site/troubleshooting.html#weave-status).  To disable this
```

Comment:
AFAIK this should reference the .md instead of the .html - all other references do so.